### ISSUE TITLE
5.0.0

### ISSUE THREAD

**Issue by tomchentw** [tomchentw/react-google-maps#267]
* [x] #224
* [x] #268
* [x] #269
* [x] #270 
* [x] #271 
* [x] #274
* [x] #300
* [x] #317

**Comment by cristiandley** [comment]
@tomchentw in this release we should also map a project tree, that way we can be sure no one is going to publish new features in the wrong path or project structure.

I will commit a readme PR on monday with the idea. I will be traveling this weekend.

**Comment by tomchentw** [comment]
@cristiandley Got it. A correct project tree is a good idea. Thanks for doing this! 👍

**Comment by cristiandley** [comment]
@tomchentw can you explain me ` Remove <GoogleMap> ` ? Im a bit lost. I want to work on this.

**Comment by rewop** [comment]
@cristiandley I think @tomchentw means that we remove the [old api to load the map](https://github.com/tomchentw/react-google-maps/blob/master/src/async/ScriptjsGoogleMap.js#L22). Am I right?

**Comment by rewop** [comment]
And also [this deprecation](https://github.com/tomchentw/react-google-maps/blob/master/src/GoogleMap.js#L89)

**Comment by tomchentw** [comment]
@rewop yes, you got it. Thanks for posting these for me. Basically, everything should be under `<XXXLoader>` in `5.0.0` and `<GoogleMap>` should be just a component responsible for maps instance.

@cristiandley go ahead, create a ticket and a PR to work on it 👍

**Comment by tomchentw** [comment]
@wuct @cristiandley @rewop anyone interested in the last task for `v5.0.0`? It's just

> Remove <GoogleMap>

**Comment by tomchentw** [comment]
Released [v5.0.0](https://github.com/tomchentw/react-google-maps/releases/tag/v5.0.0). Please notice that it's in **`next` tag on `npm`**. Install it via

```sh
npm install react-google-maps@next
# or
npm install react-google-maps@5.0.0
```

This version(`5.0.0`) will never be tagged `latest` on `npm` since there's a major rewrite ongoing for **`6.0.0`**. See #318
